packages/file-export/fastfetchbot_file_export/transcribe.py

```
@@ -0,0 +1,114 @@
+        with open(segment_path, "rb") as f:
+            result = client.audio.transcriptions.create(
+                model=TRANSCRIBE_MODEL, file=f
+            )
+            transcript += result.text
+
+        os.remove(segment_path)
+
+    os.remove(audio_file)
+    transcript = punctuation_assistant(client, transcript)
+    transcript = (
+        f"全文总结：\n{summary_assistant(client, transcript)}\n原文：\n{transcript}"
```

Ruff (0.15.1)

[warning] String contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF001)